apollo-compiler/src/test/graphql/com/example/EpisodeExpected.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public enum Episode {
```

Comment:
should we make the enum also an inner class to the query? @martijnwalraven what do you think?
Also would be nice to use the `description` field from the IR to add javadocs to the enum constants